AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -513,9 +513,7 @@ - (void)checkLatestRelease:(NSString *)updateToken distributionGroupId:(NSString
                                       @"3. The app is running in a non-adhoc environment. "
                                       @"Detach the debugger and restart the app and/or run the app with the release configuration "
                                       @"to enable the feature.");
-    @synchronized(self) {
-      self.updateFlowInProgress = NO;
-    }
+    self.updateFlowInProgress = NO;
```

Comment:
Did you have a deadlock?

Reply:
No, I didn't. I'm changing the property to atomic.